### PR TITLE
Add domain header to properties in logout

### PR DESCRIPTION
**Pull request by fazlan-nazeem**
## Purpose

Use the appropriate AuthCodeLogin method depending on whether the domain header was received.

### CODE REVIEW

File: components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java [hunk 1]
```diff
@@ -28,6 +28,7 @@
 import org.slf4j.LoggerFactory;
 import org.wso2.carbon.analytics.auth.rest.api.LoginApiService;
 import org.wso2.carbon.analytics.auth.rest.api.NotFoundException;
+import org.wso2.carbon.analytics.auth.rest.api.StringUtil;
```
Comment (TanyaM) [inline components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java]: unused import?

File: components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java [hunk 2]
```diff
@@ -43,6 +44,7 @@
 import org.wso2.carbon.analytics.idp.client.external.ExternalIdPClientConstants;
 import org.wso2.carbon.config.ConfigurationException;
 import org.wso2.carbon.streaming.integrator.common.utils.SPConstants;
+import org.wso2.carbon.utils.StringUtils;
```
Comment (TanyaM) [inline components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java]: unused import?

File: components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java (hunk 1 above)
Comment (fazlan-nazeem) [reply components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java]: Fixed

File: components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java (hunk 2 above)
Comment (fazlan-nazeem) [reply components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java]: fixed

**Pull request merged**